ydb/core/cms/cluster_info.cpp

```diff
@@ -29,20 +29,20 @@ using namespace NKikimrCms;
 bool TLockableItem::IsLocked(TErrorInfo &error, TDuration defaultRetryTime,
                              TInstant now, TDuration duration) const
 {
-    if (State == RESTART) {
-        Y_ABORT_UNLESS(Lock.Defined());
+    if (State == RESTART && HasSameOrHigherPriorityLock(Locks, PriorityToCheck)) {
+        Y_ABORT_UNLESS(!Locks.empty());
         error.Code = TStatus::DISALLOW_TEMP;
         error.Reason = Sprintf("%s is restarting (permission %s owned by %s)",
-                               PrettyItemName().data(), Lock->PermissionId.data(), Lock->Owner.data());
-        error.Deadline = Lock->ActionDeadline;
+                               PrettyItemName().data(), Locks.begin()->PermissionId.data(), Locks.begin()->Owner.data());
+        error.Deadline = Locks.begin()->ActionDeadline;
         return true;
     }
 
-    if (Lock.Defined()) {
+    if (HasSameOrHigherPriorityLock(Locks, PriorityToCheck)) {
```

Comment:
А тут `Y_ABORT_UNLESS(!Locks.empty());` не добавил.
```suggestion
    if (HasSameOrHigherPriorityLock(Locks, PriorityToCheck)) {
        Y_ABORT_UNLESS(!Locks.empty());
```